templates/bamboo/0/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+bamboo-data:
+    image: busybox
+    net: none
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+        - /home/bamboo
+
+bamboo-server:
+    image: matisq/bamboo-server:latest
+    ports:
+        - ${http_port}:8085
+        - ${agent_port}:54663
+    links:
+        - postgres
+    environment:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+        BAMBOO_VERSION: ${bamboo_version}
+        BAMBOO_BAMBOO_SERVER_ID: bamboo-server
+    labels:
+        io.rancher.sidekicks: bamboo-data
+    volumes_from:
+        - bamboo-data
+
+postgres-data:
+    image: busybox
+    net: none
+    labels:
+      io.rancher.container.start_once: true
+    volumes:
+        - /var/lib/postgresql/data/pgdata
+
+postgres:
+    image: postgres:latest
+    ports:
+        - ${postgress_port}:5432
```

Comment:
Do you really need to expose postgres port outside rancher??
Please use rancher lb's instead if it's necessary.